src/pages/resolve/components/ElectionsTable.vue

```diff
@@ -29,10 +29,18 @@
                         <q-td key="ballot_name" class="ballot_name">
                             <div class="info">
                                 <div class="ballot-name">
+                                  <a v-if="props.row.ballot_name" :href="`/trails/ballot/${props.row.ballot_name}`">
                                     {{
                                         props.row.ballot_name ||
```

Comment:
```suggestion
                                        props.row.ballot_name 
```
a tag is only displayed if `props.row.ballot_name` exists so no need for condition